### ISSUE TITLE
ServiceAuthorization config example correction

### ISSUE THREAD

**Issue by MaxWilson** [dotnet/docs#5666]
The config example currently has serviceAuthorization config element directly under system.serviceModel/behaviors. This is incorrect according to the documentation here (https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/serviceauthorization-element) and also generates design-time errors. It should be at system.serviceModel/behaviors/serviceBehaviors/behavior/serviceAuthorization.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 03c886fb-4abc-88cc-1831-74a3241c59c5
* Version Independent ID: 6933719f-0398-d9a8-57df-39fe09b40ae3
* Content: [How to: Create a Custom Authorization Policy](https://docs.microsoft.com/en-us/dotnet/framework/wcf/extending/how-to-create-a-custom-authorization-policy)
* Content Source: [docs/framework/wcf/extending/how-to-create-a-custom-authorization-policy.md](https://github.com/dotnet/docs/blob/master/docs/framework/wcf/extending/how-to-create-a-custom-authorization-policy.md)
* Service: **unspecified**
* Product: **dotnet-framework**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by mairaw** [comment]
@zhenlan can you or your team look at this issue?

**Comment by zhenlan** [comment]
@jcondex can you please confirm?

**Comment by mairaw** [comment]
ping @jcondex

**Comment by zhenlan** [comment]
Add @Lxiamail

**Comment by SubasishMohapatra** [comment]
Why is the state parameter always returning null thereby getting all claims every time from database?
public bool Evaluate(EvaluationContext evaluationContext, ref object state)

**Comment by Lxiamail** [comment]
@MaxWilson Yes, you are right, the serviceAuthorization element should be system.serviceModel/behaviors/serviceBehaviors/behavior/serviceAuthorization
